complaint_search/views.py

```diff
@@ -7,7 +7,7 @@
 from datetime import datetime
 from elasticsearch import TransportError
 import es_interface
-from complaint_search.renderers import CSVRenderer, XLSRenderer, XLSXRenderer
+from complaint_search.renderers import DefaultRenderer, CSVRenderer, XLSRenderer, XLSXRenderer
```

Comment:
Do we also need to import a separate JSON renderer, or will the above code automatically take care of the switch between export and search JSON?

Reply:
@sephcoster JSONRenderer is imported on line 3, it is part of the rest framework.  The line on 26 will automoatically take care of the switch.  So what is happening there is that the first on the list will always be a default, in this case if format is not listed, it will use this first DefaultRenderer which means `format = default`.  And it just happens that DefaultRenderer is using everything from JSONRenderer in our case.  If it is export, we will set `format = JSON` in our query parameter, it will actually route to JSONRenderer, and this is handled differently than DefaultRenderer.

Reply:
👍